algorithm.ipynb

```diff
@@ -569,10 +583,10 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.7.3"
+   "version": "3.7.1"
```

Comment:
Daily reminder of how much I hate diffing notebooks metadata.
Just for curiosity, how do you run the notebook? Is it from a virtual env?

Reply:
I think this might be my python version? I'm using pycharm to modify the file locally

Reply:
It's python. No biggie as long as the major/minor is the same :). Make sense that it diffs if you run it locally. As long as these differences happen on your box and not across stats  it's no problem at all.